pallets/msa/src/benchmarking.rs

```diff
@@ -60,7 +60,7 @@ fn create_account_with_msa_id<T: Config>(n: u32) -> (T::AccountId, MessageSource
 
 	assert_ok!(Msa::<T>::create(RawOrigin::Signed(provider.clone()).into()));
 
-	let msa_id = Msa::<T>::try_get_msa_from_public_key(&provider).unwrap();
+	let msa_id = Msa::<T>::ensure_valid_msa_key(&provider).unwrap();
```

Comment:
`try_get_msa_from_public_key` was identical to `ensure_valid_msa_key` and used in the same way.